clients/unshaded/src/main/java/tachyon/client/file/BaseFileSystem.java

```diff
@@ -49,12 +47,14 @@
 */
 @PublicApi
 public class BaseFileSystem implements FileSystem {
+  private FileSystemContext mContext;
```

Comment:
This is for the sake of possible future refactoring. IDEs like IntelliJ would be able to change the variable name if the class name changes. I am fine either way.